examples/neural_compressor/text-generation/requirements.txt

```diff
@@ -1,3 +1,4 @@
 sentencepiece != 0.1.92
 protobuf
 torch >= 2.0.0
+transformers == 4.27.4
```

Comment:
now neural engine only supports gptj and llama on transfoermers 4.27, it will crash on a higher version. It is a neural engine limitation, that will be fixed in the future.